src/index.ts

```diff
@@ -30,34 +30,48 @@ export {
     getEndDateOfRange,
     getMonth,
     getStartDateOfRange,
+    getUTCDate,
     getYear,
 } from './commonControls/dateRangeControl/DateHelper';
 export {
```

Comment:
do you think it would be easier to export * instead of list items one by one. Thus when we have new item added to the module, we won't forget to export them. And for internal use functions, we could move them to a helper file.

Reply:
yes I'm fine with export * if we are careful to put internal functions elsewhere. That seems like a reasonable thing to enforce & catch in CR via the _ convention.